prediction/src/preprocess/preprocess_ct.py

```diff
@@ -52,61 +59,81 @@ def __init__(self, hu_transform=False, clip_lower=None, clip_upper=None,
             raise ValueError('The min_max_normalize should be bool or int')
         self.min_max_normalize = min_max_normalize
 
+        if not isinstance(scale, (int, float)) and (scale is not None):
+            raise ValueError('The scale should be float or int')
```

Comment:
``TypeError`` would seem more suitable here.

Reply:
Indeed it is.